worldcup/src/components/random-result.jsx

```diff
@@ -0,0 +1,26 @@
+import { Question, Answer, AnswerImg, Button } from "../style/item";
+import { Stage } from "../style";
+import { RESULT_LIST } from "../assets/result-item";
+import Random from "./random";
+import { useState } from "react";
+
+function RandomResult(){
+    const [initial, setInitial] = useState(false);
+
+    const randomVal = Math.floor( Math.random()*RESULT_LIST.length );
+    const tempInfo = RESULT_LIST[randomVal];
+    const {link, description , ...indexs} = tempInfo;
```

Comment:
```suggestion
    const {link, description , ...indexs} = RESULT_LIST[randomVal];
```
tempInfo라는 아주잠깐! 사용되는 변수에 메모리 자원을 할당하는 것보다, 이렇게 바로 할당해주는게 더 좋겠죠?!